src/CBT.Core/Internal/ModulePropertyGenerator.cs

```diff
@@ -12,8 +12,10 @@ namespace CBT.Core.Internal
 {
     internal sealed class ModulePropertyGenerator
     {
-        internal static readonly string ImportRelativePath = Path.Combine("CBT", "Module", "$(MSBuildThisFile)");
-        internal static readonly string ModuleConfigPath = Path.Combine("CBT", "Module", "module.config");
+        internal static readonly string ImportRelativePath = Path.Combine("build", "$(MSBuildThisFile)");

```

Comment:
no this file that is generated is about importing the build.props extensions.  It has nothing to do with the build packages but extending build.props.  
What you are thinking about is NuGetBuildPackages.props which is created as part of the cbt.nuget module.